docs/en/pid-data-model.rst

```diff
@@ -0,0 +1,393 @@
+
+.. include:: ../common/common_definitions.rst
+
+.. _pid_data_model.rst:
+
+PID Data Model
+++++++++++++++
+
+The Person Identification Data (PID) is issued following national law by the PID Provider and allows a natural person to be identified. 
+
+The User attributes carried in the Italian PID are:
```

Comment:
I see your point, but here we say maybe we can add Italian because we have one main difference, in comparison with PID Rule book mandatory claims, which was age_over_nn or age_over_18

Reply:
yes, our proposal for PID is slightly different form the one in ARF/PID rulebook. As we already discussed the main reasons are:
- the claims age_over_* are qualifying adjectives and not identifying attributes
- I prefer to keep the PID dataset as small as possible to avoid excessive exposure of user data (we know that the PID widely used in the online authentication and even if the SD is enabled I prefer to minimize the risk to share more than the needed attributes.)
- It is not clear how to manage some special cases (for example, where the user asks for a PID just before his birthday (the expiration time should be given according to that date, effectively revealing the user's date of birth.)